pycqed/measurement/multi_qubit_module.py

```diff
@@ -2291,7 +2291,7 @@ def calibrate_n_qubits(qubits, f_LO, sweep_points_dict, sweep_params=None,
 
 def measure_chevron(dev, qbc, qbt, hard_sweep_params, soft_sweep_params,
                     cz_pulse_name, upload=True, label=None, qbr=None,
-                    classified=False, n_cal_points_per_state=2,
+                    classified=False, n_cal_points_per_state=1, for_ef=True,
```

Comment:
I think the `for_ef` parameter name is slightly confusing as it just adds f-level calibration points, but still conducts the standard chevron measurement, whereas in other measurement functions it also changes the experiment sequence. Can't the functionality just be acheived by setting the default `cal_states` to `'gef'`?

Reply:
What do you mean "it also changes the experiment sequence"? Here it does so too right, because it adds extra calibration segments.
I think for_ef flag has always been used as a convenience flag for the user, and yes it is equivalent to setting cal_states = 'gef'.

Reply:
Does this mean that no code changes at all would have been necessary to get the three projections of the chevrons? Then we could of course also revert these changes and just change cal_states in the notebook? Or do we want the convenience flag? @antsr @stephlazar  I do not have strong opinion here.

In the new parallelization framework, the convenience flag for_ef exists automatically for all measurements.

Reply:
I also don't mind the convenience parameter, but I was just expecting it to do something else at first sight. Does this parameter just add the f-level calpoints in `the new parallelization framework`?

If it is just about the cal-points and nothing about the actual experiment, then I would slightly prefer to use cal_states and omit the for_ef parameter. 

You can make the call @stephlazar what makes more sense to you.

Reply:
Yes, I agree we should clean up this part of the code. I also think we should get rid of it, especially because now we are considering more and more using h or higher levels so then for_ef is a bit pointless. 
@chellings in the new framework I think the user can specify cal_states as well, so we should remove for_ef. 
But to answer your previous question @chellings, indeed no code changes at all would have been necessary to get the three projections of the chevrons. I just tested it on another branch which does not have these changes and it worked if I passed cal_states='gef'. So I think this pull request is in fact not needed. Except maybe the n_cal_points_per_state=1 change.

Reply:
So do you want to change the defaults to cal_states=('g', 'e', 'f') and n_cal_points_per_state=1? Or should we leave everything as it was and I close the pull request? @antsr @stephlazar

Reply:
I would do that. Change the defaults to cal_states=('g', 'e', 'f') and n_cal_points_per_state=1